my/src/components/Card.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <div class="container">
-      <h1 class="question" id="title1">{{ title }}</h1>
+      <h1 class="question" v-html="title"></h1>
```

Comment:
Our questions are now parsed as HTML, which was p much needed for Q3. It's not always a good idea to do this b/c XSS, but that doesn't really apply here